components/region_cache_memory_engine/src/lib.rs

```diff
@@ -52,6 +54,7 @@ impl Default for RangeCacheEngineConfig {
         Self {
             enabled: false,
             gc_interval: ReadableDuration(Duration::from_secs(180)),
+            load_evict_interval: ReadableDuration(Duration::from_secs(360)),
```

Comment:
Please add comments to explain why 360s here. Like assuming there are 100K regions, the computing of loading & evicting is ...